add-on/chrome/modules/MozLoopService.jsm

```diff
@@ -1312,11 +1313,30 @@ this.MozLoopService = {
 
       let window = gWM.getMostRecentWindow("navigator:browser");
       if (window) {
+        let isOwnerInRoom = false;
+        // The participant that joined isn't necessarily included in room.participants (depending on
+        // when the broadcast happens) so concatenate.
+        for (let roomParticipant of room.participants.concat(participant)) {
+          if (roomParticipant.owner) {
+            isOwnerInRoom = true;
+            break;
+          }
+        }
+
+        let localizedString;
+        if (isOwnerInRoom) {
+          localizedString =
+            MozLoopServiceInternal.localizedStrings.get("rooms_room_joined_owner_connected_label2");
+        } else {
+          let l10nString = MozLoopServiceInternal.localizedStrings.get("rooms_room_joined_owner_not_connected_label");
+          let roomUrlDomain = new URL(room.decryptedContext.urls[0].location).hostname;
+          localizedString = PluralForm.get(1, l10nString).replace("#url", roomUrlDomain);
```

Comment:
I'm afraid you're confusing string substitution as plural form; PluralForm.jsm should be used when there are different versions of a string depending on the number of subjects. Read all the gritty at https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals#Developing_with_PluralForm (worth the read :wink: )

For my proposal, see my comment at shared.properties.